cypress/support/Contentful/Models/SeriesModel.js

```diff
@@ -37,11 +37,15 @@ export class SeriesModel {
     }
 
     get startDate(){
-        return this.starts_at.split("T")[0]; //Remove time zone
+        if(this.starts_at !== undefined)
```

Comment:
could you use a ternary here?
```suggestion
return this.starts_at !== undefined ? this.starts_at.split("T")[0] : this.starts_at
```